infrastructure/controllers/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- cert-manager.yaml
```

Comment:
some resources are in the same directory, but missing here:
- `external-dns.yaml`
- `metallb.yaml`